server/src/main/scala/net/wiringbits/repositories/daos/TablesDAO.scala

```diff
@@ -0,0 +1,19 @@
+package net.wiringbits.repositories.daos
+
+import anorm.SqlStringInterpolation
+import net.wiringbits.repositories.models.Table
+
+import java.sql.Connection
+
+object TablesDAO {
+
+  def all()(implicit conn: Connection): List[Table] = {
+    SQL"""
+        SELECT table_name
+          FROM information_schema.tables
+        WHERE table_schema='public'
```

Comment:
Be aware that this is not generic enough, you do not need to look for a solution but we need to add a `TODO` to fix it some time